src/VoxaApp.ts

```diff
@@ -542,13 +539,13 @@ export class VoxaApp {
   }
 
   public async transformRequest(voxaEvent: IVoxaEvent): Promise<void> {
-    await this.i18nextPromise;
+    await initializeI118n(this.config.views);
     const data = voxaEvent.session.attributes.model as IBag;
     const model = await this.config.Model.deserialize(data, voxaEvent);
 
     voxaEvent.model = model;
     voxaEvent.log.debug("Initialized model ", { model: voxaEvent.model });
-    voxaEvent.t = this.i18n.getFixedT(voxaEvent.request.locale);
+    voxaEvent.t = i18n.getFixedT(voxaEvent.request.locale);
```

Comment:
are we sure this is correct? Before this change there was a new instance of i18n created for every request. Seems like this change will make it so there's only one global instance. In the past this was an issue for multi language skills, a german skill would get the english translations from time to time, or viceversa

@omenocal